### ISSUE TITLE
Project abandoned?

### ISSUE THREAD

**Issue by nama17** [goryn-clade/pathfinder#241]
I've been told this project is/will be abandoned soon, can anyone confirm or deny this? If so, is there another "official" fork that's taking over?

**Comment by TyrHeimdalEVE** [comment]
No, it's not abandoned. However it's not under active development in terms of features either.

We do however intend to fix any issues that arise however, to keep it usable.

Outside of that PR's are welcome, but might not get into a release until something bigger has to be changed unless it's critical to everyday usage.
